code/style.css

```diff
@@ -1,4 +1,208 @@
-html {
-  background: #0025ff;
-  color: #fff;
+*, *::before, *::after {box-sizing: border-box;}
+
+body {
+  /***background: #a3a37d;***/
+  color: #727768;
+  font-family:'Open Sans Condensed', sans-serif;
+  margin: 0px;
 }
+
+/* HEADER+NAVBAR*/
+
+header {
+  background: white;
+  height: 100px;
+}
+
+header:hover {
+  background:#e5e6b4;
+  opacity: 0.5;
+  transition: all 500ms ease;
+  
+}
+
+nav a  {
+  text-decoration: none;
+  color: #727768;
+  margin: 10px;
+  padding: 10px 10px;
+  font-size: 20px;
+  float: right;
+}
+
+h1 {
+  font-size: 60px;
+}
+
+/* INLEDNING*/
+
+.big-news {
+  font-size: 20px;
+  margin-right: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  flex-direction: column;
+  width: 100%;
+  padding-top: 40px;
+  padding-bottom: 80px;
+
+}
+
+.big-news img {
+  width: 100%;
+  max-width: 100%;
+}
+
+.big-news div {
+  text-align: center;
+  padding: 25px;
+}
+
+/* CONTAINERS*/
+
+.plants {
+  background:  #d0d8c1;
+  padding: 25px;
+}
+.plants > h4 {
+  text-align: center;
+  margin: 70px 0px 70px 0px;
+  font-size: 20px;
+  border: 1px solid #a3a37d;
+  padding: 15px;
+}
+
+.plant-containers {
+  display: flex;
+  flex-wrap: wrap;
+  /*justify-content: center;*/
+}
+
+.card {
+  background: rgb(180, 192, 181);
+  text-align: center;
+  margin: 25px;
+  margin-bottom: 40px;
+  padding: 25px;  
+  border-radius: 15px;
+  box-shadow: 12px 12px 20px 0 #61725a;
+  width: 100%;
+}
+
+.card h4 {
+
+  margin:0px;
+}
+
+.plant-containers img {
+  width: 40%;
+}
+
+/*FOOTER*/
+
+footer {
+  font-size: 20px;
+  text-align: center;
+  color: #727768;
+  width: 100%;
+  height:100px;
+  margin-top: 30px;
+  position: relative;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

Comment:
Do you remember when I mentioned to all of you during one lecture that there is a specific order of these properties? The correct one should be `top`, `right`, `bottom`, `left`